src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -336,3 +336,86 @@ ov::pass::GeluFusionWithTanh::GeluFusionWithTanh() {
     auto m = std::make_shared<ov::pass::pattern::Matcher>(mul_3, matcher_name);
     register_matcher(m, callback);
 }
+
+ov::pass::GeluFusionWithTanhNoPower::GeluFusionWithTanhNoPower() {
+    // Replaces a sub-graph with a Gelu (ov::op::v0::Tanh) op
+    // x * 0.5 * (1 + tanh((x * 0.044715 * x + 1) * x * sqrt(2 / pi)))
+    MATCHER_SCOPE(GeluFusionWithTanhNoPower);
+    auto input = pattern::any_input();
+
+    auto const1 = pattern::wrap_type<ov::op::v0::Constant>();
+    auto mul1 = pattern::wrap_type<ov::op::v1::Multiply>({input, const1});
+
+    auto mul2 = pattern::wrap_type<ov::op::v1::Multiply>({mul1, input});
+
+    auto const2 = pattern::wrap_type<ov::op::v0::Constant>();
+    auto add1 = pattern::wrap_type<ov::op::v1::Add>({const2, mul2});
+
+    auto const3 = pattern::wrap_type<ov::op::v0::Constant>();
+    auto mul3 = pattern::wrap_type<ov::op::v1::Multiply>({input, const3});
+
+    auto mul4 = pattern::wrap_type<ov::op::v1::Multiply>({add1, mul3});
+
+    auto tan = pattern::wrap_type<ov::op::v0::Tanh>({mul4});
+
+    auto const4 = pattern::wrap_type<ov::op::v0::Constant>();
+    auto add2 = pattern::wrap_type<ov::op::v1::Add>({tan, const4});
+
+    auto const5 = pattern::wrap_type<ov::op::v0::Constant>();
+    auto mul5 = pattern::wrap_type<ov::op::v1::Multiply>({input, const5});
+
+    auto mul6 = pattern::wrap_type<ov::op::v1::Multiply>({add2, mul5});
+
+    matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto& pattern_to_output = m.get_pattern_value_map();
+        auto x_output = pattern_to_output.at(input);
+
+        auto const1_value =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_to_output.at(const1).get_node_shared_ptr());
+        auto const2_value =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_to_output.at(const2).get_node_shared_ptr());
+        auto const3_value =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_to_output.at(const3).get_node_shared_ptr());
+        auto const4_value =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_to_output.at(const4).get_node_shared_ptr());
+        auto const5_value =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_to_output.at(const5).get_node_shared_ptr());
+
+        if (!const1_value || !const2_value || !const3_value || !const4_value || !const5_value) {
+            return false;
+        }
+
+        constexpr float pi = 3.14159265358979323846;
```

Comment:
the comment from Ilya Lavrenov was not fixed
is it possible to use this constant from "cmath"?